test/solidity/Periphery/TokenWrapper.t.sol

```diff
@@ -51,4 +96,121 @@ contract TokenWrapperTest is DSTest {
         tokenWrapper.withdraw();
         assert(address(this).balance - initialBalance == 1 ether);
     }
+
+    // ========== CONVERTER TESTS ==========
+
+    function testCanDepositWithConverter() public {
```

Comment:
all tests are not named according to our naming convention.....🫩
Please also update existing (untouched) tests to match our conventions.